qosc/OscSender.cpp

```diff
@@ -71,13 +71,14 @@ void OscSender::send(const QString& oscAddress, const QVariantList& arguments) {
 }
 
 
-void OscSender::variantListToByteArray(QByteArray& outputResult, const QString& oscAddress, const QVariantList& arguments) {
+void OscSender::variantListToByteArray(QByteArray& outputResult, const QVariantList& arguments) {
```

Comment:
Why are you modifying qosc/OscSender.cpp? I believe this is not required, since it's a generic class that we can reuse from project to project. (I wrote most of it) 

Here, removing the oscAdress argument - which is a path - is probably not a good idea: how does one send and OSC message without a path? Paths are part of the OSC specifications.

Therefore, unless there is a good reason to remove the oscAddress argument from OscSender:: variantListToByteArray and OscSender::send, I would advocate for avoiding to remove it.

Reply:
“/GeneratorName/output” should not be passed as an argument
It should be the OSC path. (the oscAddress argument to the OscSender::send method)
The oscAddress argument of the OscSender::send method should not be removed.

Reply:
I agree! This code was written this way by someone else, previous to my involvement on the project. This task could be turned into a new ticket for the next round of work - and these changes left behind as a draft / note for the person who does that work.